internal/usecases/user/usecase.go

```diff
@@ -34,37 +34,22 @@ type Usecases struct {
 }
 
 // Create Add new user
-func (uc *Usecases) Create(ctx context.Context, nu entity.NewUser) (string, error) {
-	hash, err := bcrypt.GenerateFromPassword([]byte(nu.Password), bcrypt.DefaultCost)
+func (uc *Usecases) Create(ctx context.Context, u *entity.User) error {
+	hash, err := bcrypt.GenerateFromPassword([]byte(u.PasswordHash), bcrypt.DefaultCost)
```

Comment:
generate hasp from user.passwordHash - is not straitforward to understand)))

Reply:
any suggestion ?

Reply:
for now, we could pass userRequest

Reply:
but if we use not http something it will be hard